stable/rabbitmq/values.yaml

```diff
@@ -349,6 +349,22 @@ metrics:
     prometheus.io/scrape: "true"
     prometheus.io/port: "9090"
 
+  livenessProbe:
+    enabled: true
```

Comment:
It doesn't look like this `enabled` field is used anywhere. I would wrap the liveness and readiness probes definitions in the Stateful Set with an `if`, just like the main container probes, to allow enabling and disabling them.

Reply:
Good catch - updated to make use of `enabled` flags in dfcf104200844279ec03f7c4ee199419cb102297